sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/SparkSQLCLIDriver.scala

```diff
@@ -278,8 +278,16 @@ private[hive] object SparkSQLCLIDriver extends Logging {
 
     var ret = 0
     var prefix = ""
-    val currentDB = ReflectionUtils.invokeStatic(classOf[CliDriver], "getFormattedDb",
-      classOf[HiveConf] -> conf, classOf[CliSessionState] -> sessionState)
+
+    def currentDB: String = {
+      val printCurrentDb = conf.getBoolVar(HiveConf.ConfVars.CLIPRINTCURRENTDB) ||
```

Comment:
so there is already a hive conf to control this behavior?

Reply:
Inherit the behavior from `spark.sql.cli.print.header/hive.cli.print.header`. A dynamic sql conf may be convenient for using set syntax together.